google/resource_google_folder.go

```diff
@@ -63,7 +63,13 @@ func resourceGoogleFolderCreate(d *schema.ResourceData, meta interface{}) error
 		return fmt.Errorf("Error creating folder '%s' in '%s': %s", displayName, parent, err)
 	}
 
-	err = resourceManagerV2Beta1OperationWait(config.clientResourceManager, op, "creating folder")
+	opV1 := make(map[string]interface{})
+	waitErr := Convert(op, opV1)
```

Comment:
@rambleraptor You can't use a `map` in convert, you have to pass a struct. Convert is a bit brittle